apps/web/package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "scripts": {
     "clean": "rimraf .turbo node_modules .next",
-    "dev": "next dev -p 3000",
-    "go": "next dev -p 3000",
+    "dev": "next dev --experimental-https -p 3000",
+    "go": "next dev --experimental-https -p 3000",
```

Comment:
Using experimental https for local development enforces getting the certificates and changes in the .env file(`WEBAPP_URL` & `NEXTAUTH_URL`) and changes in the redirect_url of other integrations. Can we mention in the docs about the https-only mode in case of slack integration?
what do you think? @mattinannt

Reply:
Maybe this is only relevant for the development & testing of the integration and we can remove them in the PR itself before merging. Later we can see if it maybe makes sense to by default use https mode for the dev server when the feature is not experimental any more.
I don't think we need to document this for now.